src/main/java/com/example/demo/model/User.java

```diff
@@ -37,12 +40,13 @@ public class User implements UserDetails {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
don't use Eager fetchtype

```suggestion
    @ManyToMany
```